src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <div className="Person">
+      <p className="Person__name">
+        {`My name is ${name}`}
+      </p>
+      {age !== undefined && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+         {partnerName} is my ${partner}
```

Comment:
I can't push my commit anymore somehow, so there is my code. I'm gonna contact my local mentor on QA to resolve this problem
<p className="Person__partner">
        {isMarried ? `${partnerName} is my ${partner}` : 'I am not married'}
      </p>

Reply:
Fixed